### ISSUE TITLE
[java] False positive JUnitTestContainsTooManyAsserts for JUnit 5 assertAll()

### ISSUE THREAD

**Issue by christianhujer** [pmd/pmd#2374]
**Affects PMD Version:** 6.22.0

**Rule:** JUnitTestContainsTooManyAsserts

**Description:**
When `Assertions.assertAll()` is used, that should be treated as a single assert.

**Code Sample demonstrating the issue:**

```
class SampleTest {
    @Test
    void sampleTestCase() {
        final Point input = new Point(1, 2);
        final Point result = p.add(p);
        assertAll(
            () -> assertEquals(2, result.x),
            () -> assertEquals(4, result.y),
        );
    }
}
```

**Running PMD through:** *Gradle*

**Comment by oowekyala** [comment]
Duplicate of #1212
